include/openmc/source.h

```diff
@@ -19,32 +19,47 @@ namespace openmc {
 // Global variables
 //==============================================================================
 
-class SourceDistribution;
+class Source;
 
 namespace model {
 
-extern std::vector<SourceDistribution> external_sources;
+extern std::vector<std::unique_ptr<Source>> external_sources;
```

Comment:
Both before and now, any of these would have been considered external sources (as opposed to a fission source in a k-eigenvalue calculation that is not known a priori). I think I chose `external_sources` originally to avoid potentially overlapping terminology (e.g., "source" as a programming concept), but I'm open to renaming.

Reply:
I think that makes good sense. No need to discuss renaming. I just wasn't thinking of it that way.